publisher-sdk/proguard-rules.pro

```diff
@@ -21,12 +21,9 @@
 #-renamesourcefileattribute SourceFile
 # Retrofit does reflection on generic parameters. InnerClasses is required to use Signature and
```

Comment:
Nit: you might update the documentation to indicate that, as an open source project, we keep most info and we obfuscate only to optimize the public API

Also, I guess we can remove all the generated doc above. It doesn't bring any value to this file.

Reply:
Done.